### DISCUSSION TITLE
Custom Fields returns values in GraphiQL IDE but `null` in Next.js

### DISCUSSION THREAD

**Discussion by marco910** [wp-graphql/wp-graphql#2682] [Advanced Custom Fields]
Hi there,

I'm using WPGraphQL with my Next.js app. Everything works fine when trying to query custom ACF field groups expect one thing. When trying to query a select field, where the author can select users from the WP site, the field returns the correct values when querying inside GraphiQL IDE on WordPress. But if I query from Next.js (with Apollo Client) I'm getting a `null` value for each field.
I'm able to query all other fields. Just those two ones are not working.

Any ideas what this could cause?

**Comment by justlevine** [comment]
Most User data is private to authenticated requests.

Sounds like you're running your queries as an authenticated user in the IDE (the green dot on your profile image), but unauthenticated in Next.JS

**Reply by marco910** [reply]
Thanks @justlevine !
The issue was that the default `author` user object is always public in this case, but not so other user information. I was querying in the IDE as logged-in user, so it worked there. After adding authentication to WordPress to my Apollo Client, it's also working now in Next.js
